src/PerfCounters.cc

```diff
@@ -302,6 +319,12 @@ void PerfCounters::reset(Ticks ticks_period) {
     attr.config |= IN_TXCP;
     fd_ticks_measure = start_counter(tid, fd_ticks_interrupt, &attr);
 
+    if (activate_useless_counter && !fd_useless_counter.is_open()) {
+      // N.B.: This is deliberately not in the same group as the other counters
+      // since we want to keep it scheduled at all times.
+      fd_useless_counter = start_counter(tid, -1, &cycles_attr);
```

Comment:
This one's probably fine, since it should be able to use the fixed function counter on most microarchitectures, so there shouldn't be much of a penalty, if any.